grpc/doc/swdesign/README.md

```diff
@@ -252,6 +256,23 @@ Needs:
 - impl
 - itest
 
+#### gRPC Client send supported version with first message
+`swdd~grpc-client-sends-supported-version~1`
+
+Status: approved
+
+The gRPC Client shall send the Ankaios version it was build with in the first message to the gRPC server.
```

Comment:
Well .. after the decision that server and agent shall be written small, Maybe we should change to gRPC client instead of the suggested capitalized Server.